### PR TITLE
List AWS regions added in release 20160813014741

### PR DESCRIPTION
**Pull request by ianmalott**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by ianmalott** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;hItLQZNx&#x2F;562-update-supported-aws-regions-doc"> Update supported AWS regions doc</a>

File: _reference/supported-aws-regions.md [hunk 1]
```diff
@@ -6,14 +6,17 @@ Convox will run in any region that supports EC2 Container Service, Lambda and at
...
+- ap-northeast-1
+- ap-southeast-1
 - ap-southeast-2
+- eu-central-1
 - eu-west-1
 - us-east-1
 - us-west-2
 
 
 ## Future Inclusion
```
Comment (ddollar) [inline _reference/supported-aws-regions.md]: Can just remove this whole section now.

Comment (ianmalott) [inline _reference/supported-aws-regions.md]: Done 👍

**Comment by nzoschke** [comment]
LGTM! I'll be referencing this from the newsletter.

**Pull request merged**